### ISSUE TITLE
Issues with striked lines and lists

### ISSUE THREAD

**Issue by abpaul** [blocknotes/activeadmin_trumbowyg#3]
Hello!

I'm having some issues with striked lines and lists.
I can't undo the effect of the striked line and lists do not appear correctly in the editor. I've added a screenshot as an example. 

![example](https://user-images.githubusercontent.com/33491362/33291761-3ce614a8-d3c7-11e7-8977-8303d5124f4b.png)

As you can see, most of the text is correctly rendered but the lists do not have bullets or numbers and the striked line doesn't revert by clicking again on the button, but it works if I use ctrl+z

**Comment by blocknotes** [comment]
Hi @paulbandrei 
**List style problem**: I checked with 2 projects (AA 1.1.0) that I use as test installations and the list seems ok. One project without theme and the other one with my theme (Blaze). I would suggest you to try to inspect the list to see where is the style override (the UL should have `list-style-type: disc` instead of `none`).
**Deleted text problem**: confirmed. Same problem here. I tried to update the editor but it remains. I found an old closed issue (on Trumbowyg editor project) and I wrote a [comment](https://github.com/Alex-D/Trumbowyg/issues/293#issuecomment-348751098) there. The problem is also on their demo page

**Comment by abpaul** [comment]
@blocknotes sorry for not actually updating this issue.

**List style problem**: A day after I've posted this issue I decided to investigate a bit on this bug on the front-end as well, I jumped on the browser inspector tool and I've seen that the list styles were not applied (as you've said, the `list-style-type` was set to `none`). This is an issue with the theme I am using, Artic. 

**Deleted text problem**: Indeed, I've also seen that issue on their GitHub page. 
Might be worth adding a note to the README?
